task_processing/metrics.py

```diff
@@ -0,0 +1,54 @@
+import mock
+
+try:
+    import yelp_meteorite
+    METRICS_ENABLED = True
+except Exception:
+    METRICS_ENABLED = False
+
+TASKPROC_TASK_LAUNCHED_COUNT = 'taskproc_task_launched_count'
```

Comment:
Was trying to follow yelp_kew's example, and yelp_meteorite's best practices. Thoughts?